src/Page/PeoplePage.tsx

```diff
@@ -0,0 +1,66 @@
+import { FC, useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { getPeople } from '../api';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/Table/PeopleTable';
+import { Person } from '../types';
+
+export const PeoplePage: FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [hasError, setHasError] = useState(false);
+  const { selectedSlug = '' } = useParams();
+
+  const loadPeople = async () => {
+    setIsLoading(true);
+
+    try {
+      const peopleFromServer = await getPeople();
+
+      setPeople(peopleFromServer);
+    } catch {
+      setHasError(true);
+    }
+  };
+
+  useEffect(() => {
+    loadPeople();
+  }, []);
+
+  useEffect(() => {
+    if (isLoading) {
+      setIsLoading(false);
+    }
+  }, [people, hasError]);
```

Comment:
remove it and `setIsLoading(false)` in finally block inside `loadPeople` function